src/pluginVanilla/java/mcp/mobius/waila/plugin/vanilla/provider/PlantProvider.java

```diff
@@ -31,6 +35,31 @@ private static void addMaturityTooltip(ITooltip tooltip, float growthValue) {
         }
     }
 
+    private static void addCropGrowableTooltip(ITooltip tooltip, IBlockAccessor accessor) {
+        int lightLevel = accessor.getWorld().getRawBrightness(accessor.getPosition(), 0);
+        tooltip.addLine(new PairComponent(
+            Component.translatable(Tl.Tooltip.CROP_GROWABLE), lightLevel >= 9
+            ? Component.translatable(Tl.Tooltip.TRUE)
+            : Component.translatable(Tl.Tooltip.FALSE)));
+    }
+
+    private static void addTreeGrowableTooltip(ITooltip tooltip, IBlockAccessor accessor) {
+        int lightLevel = accessor.getWorld().getRawBrightness(accessor.getPosition(), 0);
+        boolean mangrovePropaguleHanging;
+        boolean growable = lightLevel >= 9;
+        if (accessor.getBlock() instanceof MangrovePropaguleBlock) {
+            mangrovePropaguleHanging = accessor.getBlockState().getValue(MangrovePropaguleBlock.HANGING);
+            if (mangrovePropaguleHanging) {
+                growable = false;
+            }
```

Comment:
```suggestion
            if (accessor.getBlockState().getValue(MangrovePropaguleBlock.HANGING)) {
                growable = false;
            }
```

The `mangrovePropaguleHanging` variable should also be removed.